src/main/java/com/checkout/payments/GetPaymentResponse.java

```diff
@@ -40,7 +40,7 @@
     private List<PaymentActionSummary> actions = new ArrayList<>();
 
     public boolean requiresRedirect() {
-        return hasLink("redirect");
+        return getLinks().containsKey("redirect");
```

Comment:
Please create a private constant for the value `redirect`.

Reply:
Sure thing